examples/plot_grad_search_cv.py

```diff
@@ -12,117 +12,117 @@
 #
 # License: BSD (3-clause)
 
```

Comment:
should you revert the changes here?

Reply:
I planned to do the CV in a separated PR because this one is becoming too big